hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/util/SigReqAdapterUtils.java

```diff
@@ -82,4 +83,12 @@ public PlatformTxnAccessor platformTxn() {
                 };
         return dummyScenario.tokenStore();
     }
+
+    public static TransactionBody txnFrom(final TxnHandlingScenario scenario) {
+        try {
+            return scenario.platformTxn().getTxn();
+        } catch (final Throwable e) {
+            throw new RuntimeException(e);
```

Comment:
Can you add a message to the exception?

Reply:
Do we need that given this is test code?